web/app.js

```diff
@@ -419,10 +419,14 @@ var PDFViewerApplication = {
     var support = !!(doc.requestFullscreen || doc.mozRequestFullScreen ||
                      doc.webkitRequestFullScreen || doc.msRequestFullscreen);
 
+//#if MOZCENTRAL
```

Comment:
Just so I understand, do you want to get rid of the `!!doc.requestFullscreen` part, or did you mean something like the following?
```javascript
//#if MOZCENTRAL
//  var support = !!doc.requestFullscreen && document.fullscreenEnabled === true;
//#else
  ...
//#endif
```